ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -40,238 +41,168 @@ Item {
         spacing: 0
         interactive: height !== contentHeight
 
-        delegate: Loader {
+        delegate: DropArea {
             id: chatLoader
+            objectName: model.name
+            width: model.isCategory ? statusChatListCategoryItem.width : statusChatListItem.width
+            height: model.isCategory ? statusChatListCategoryItem.height : statusChatListItem.height
+            keys: ["x-status-draggable-list-item-internal"]
```

Comment:
I suggest using a different than the default key here, for both the delegate and the drop area

Reply:
updated